tasks/ThrusterTask.cpp

```diff
@@ -95,14 +93,13 @@ void ThrusterTask::cleanupHook()
     node->Fini();
 }
 
-void ThrusterTask::setGazeboModel(WorldPtr _world,  ModelPtr _model, sdf::ElementPtr _plugin)
+void ThrusterTask::setGazeboThruster( ModelPtr model )
```

Comment:
I mean that, in the long run, we will transition to a scheme where not all tasks have to be defined in the rock_gazebo gazebo plugin. Which means that they will have to have all the same interface. I.e. the old setGazeboModel

Do NOT change setGazeboWorld as there is only one world plugin.